### PR TITLE
Feature/5 persist locations as geo points

### LINKED ISSUE #5: Persist Locations as postgis objects
Address objects need to be converted to locations (/geopoint) service already exists for that. 

Create necessary entities to persists those locations as postgis

### PR DESCRIPTION
**Pull request by meteohr**
No lambdas were deployed yet. Only local testing.

### CODE REVIEW

File: common/help-on-spot-models/src/utils/getGeolocation.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import Lambda from "aws-sdk/clients/lambda";
+
+require('dotenv').config();
+
+export async function getPointFromGeoservice(address: AddressData): Promise<number[]> {
+  const awsConfig = new Config({
+    accessKeyId: process.env.AWS_ACCESS_KEY,
+    secretAccessKey: process.env.AWS_SECRET_KEY,
+    region: process.env.AWS_REGION
+  })
```
Comment (meteohr) [inline common/help-on-spot-models/src/utils/getGeolocation.ts:11]: Not sure if this is actually need for the execution in the lambdas itself.

File: common/help-on-spot-models/src/utils/getGeolocation.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  const lambda = new Lambda(awsConfig);
+  const params = {
+    FunctionName: 'HoS-geolocation-dev',
```
Comment (izi01) [inline common/help-on-spot-models/src/utils/getGeolocation.ts]: Setting is via ENV would be helpful, I guess

File: common/help-on-spot-models/src/utils/getGeolocation.ts [hunk 3]
```diff
@@ -2,29 +2,28 @@ import {AddressData} from "../models/RestModels";
...
+  // credentials are only needed for local setup
   const awsConfig = new Config({
     accessKeyId: process.env.AWS_ACCESS_KEY,
     secretAccessKey: process.env.AWS_SECRET_KEY,
     region: process.env.AWS_REGION
   })
   const lambda = new Lambda(awsConfig);
   const params = {
-    FunctionName: 'HoS-geolocation-dev',
+    FunctionName: process.env.GEOSERVICE_LAMBDA_NAME as string,
```
Comment (izi01) [inline common/help-on-spot-models/src/utils/getGeolocation.ts:14]: Guess this needs to be set then in every post and patch lambda?

**Comment by meteohr** [comment]
PR is not ready yet. Please don't merge.

File: common/help-on-spot-models/src/utils/getGeolocation.ts (hunk 3 above)
Comment (meteohr) [reply common/help-on-spot-models/src/utils/getGeolocation.ts:14]: I added it to every lambda, which uses gelocation service

**Pull request merged**